src/main/java/com/revature/controller/AnswerController.java

```diff
@@ -1,8 +1,29 @@
 package com.revature.controller;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.revature.entities.Answer;
+import com.revature.services.AnswerService;
+
 @RestController
+@RequestMapping("/answers")
 public class AnswerController {
+	
+	@Autowired
+	AnswerService answerService;
+	
+	@GetMapping
+	public Page<Answer>getAnswers(Pageable pageable,
```

Comment:
If we are sending a parameter in we need an if statement to control the logic of having the id passed